core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -2254,4 +2254,7 @@ CacheConfigurationException storeConfiguredHasBothReadAndWriteOnly(String storeC
    @LogMessage(level = WARN)
    @Message(value = "Failed to transfer cross-site tombstones to %s for segments %s.", id = 662)
    void failedToTransferTombstones(Address requestor, IntSet segments, @Cause Throwable t);
+
+   @Message(value = "Cache name must be shorter than 255 characters, current name '%s' exceeds the size.", id = 663)
```

Comment:
A cache name of 255 characters is also valid, so maybe we should update the message to:

""Cache name must be less than 256 characters, current ..."

Let's not worry about if the user tries to use non-ASCII chars in their cache name :slightly_smiling_face: 

We should probably update the message of the `IllegalArgumentException` thrown by the `ByteString` to "ByteString must be less than 256 bytes" as well.